<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_dialog</class>
 <widget class="QDialog" name="Edit_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>611</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="10" column="3">
     <widget class="QLineEdit" name="lineEdit_16"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>联系人</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>经费负责人</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>名称</string>
      </property>
     </widget>
    </item>
    <item row="12" column="3">
     <widget class="QLineEdit" name="lineEdit_17"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>供应商</string>
      </property>
     </widget>
    </item>
    <item row="16" column="3">
     <widget class="QDateEdit" name="dateEdit_6"/>
    </item>
    <item row="8" column="3">
     <widget class="QLineEdit" name="lineEdit_14"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>购买日期</string>
      </property>
     </widget>
    </item>
    <item row="14" column="3">
     <widget class="QLineEdit" name="lineEdit_19"/>
    </item>
    <item row="4" column="3">
     <widget class="QDateEdit" name="dateEdit_4"/>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="lineEdit_12"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>卡号</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>出厂日期</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>类型</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>验收人</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>送货日期</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLineEdit" name="lineEdit_13"/>
    </item>
    <item row="8" column="4">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>经费</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QDateEdit" name="dateEdit_5"/>
    </item>
    <item row="8" column="5">
     <widget class="QLineEdit" name="lineEdit_15"/>
    </item>
    <item row="6" column="4">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>电话</string>
      </property>
     </widget>
    </item>
    <item row="12" column="4">
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>经办人</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>经费类型</string>
      </property>
     </widget>
    </item>
    <item row="14" column="4">
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>使用人</string>
      </property>
     </widget>
    </item>
    <item row="12" column="5">
     <widget class="QLineEdit" name="lineEdit_18"/>
    </item>
    <item row="14" column="5">
     <widget class="QLineEdit" name="lineEdit_20"/>
    </item>
    <item row="16" column="5">
     <widget class="QComboBox" name="comboBox_6">
      <item>
       <property name="text">
        <string>正在使用</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>闲置</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="16" column="4">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>使用状态</string>
      </property>
     </widget>
    </item>
    <item row="19" column="4">
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>备注</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="label_36">
      <property name="text">
       <string>基本配置</string>
      </property>
     </widget>
    </item>
    <item row="19" column="3">
     <widget class="QTextEdit" name="textEdit_3"/>
    </item>
    <item row="19" column="5">
     <widget class="QTextEdit" name="textEdit_4"/>
    </item>
    <item row="0" column="5">
     <widget class="QComboBox" name="comboBox_4">
      <item>
       <property name="text">
        <string>设备</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>材料</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="10" column="5">
     <widget class="QComboBox" name="comboBox_5">
      <item>
       <property name="text">
        <string>横向</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>纵向</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>自主基金</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineEdit_11"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>设备编号</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="3">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
